tasks/setup_database.yml

```diff
@@ -1,9 +1,27 @@
 ---
 - name: ensure postgresql is ready
+  become: true
   ansible.builtin.service:
     name: postgresql
     state: started
     enabled: true
+  notify: restart psql
+
+- name: Ensure required locale is present
+  become: yes
+  community.general.locale_gen:
+    name: en_US.UTF-8
+    state: present
+  notify: restart psql
+
+- name: Run handlers immediatly
+  meta: flush_handlers
+
+- name: Wait for PostgreSQL database to become available
+  wait_for:
+    port: 5432
+    delay: 5
+    timeout: 60
```

Comment:
I believe it's out of this scope to choose the locale for the system. It's up to the user. The locale has been moved to a role variable to reflect this choice in de3d5b5.

Reply:
The reason why I choose to to install a locale is that en_US.UTF-8 was not present at least one the Azure instances I use to test.
I would like to keep this installation step but will replace static en_US.UTF-8 with the variable you introduced.

Reply:
Managing locale is not up to this role. Locale must be managed system-wise, so I suggest users to manage this on their own, hence the configurable locale with the appropriate variables.

Reply:
Okay, I see where we are going. I will remove this and the `acl` package but will put warnings into the documentation, so the user is aware of it.